### PR TITLE
[SMALLFIX]replace anonymous type with lambda in alluxio.client.file.RetryHandli…

### PR DESCRIPTION
**Pull request by Summerqqq**
…ngFileSystemMasterClient#mount

### CODE REVIEW

**Comment by gurmeetb** [comment]
@Summerqqq could you fix other RPC dependencies? I see there are various other methods which need this change in that class.

File: pom.xml [hunk 1]
```diff
@@ -784,6 +784,7 @@
   </pluginRepositories>
 
   <build>
+    <defaultGoal>compile</defaultGoal>
```
Comment (apc999) [inline pom.xml]: @Summerqqq  why we need this change?

Comment (Summerqqq) [reply pom.xml]: No goals have been specified for this build.解决方法：pom.xml文件<build>标签后面加上<defaultGoal>compile</defaultGoal>即可

**Comment by yupeng9** [comment]
@Summerqqq It seems your change was based off branch 1.6. Can you cherry-pick your change to master and recreate the PR?

**Pull request closed without merging**